hvplot/plotting/core.py

```diff
@@ -496,25 +496,10 @@ def scatter(self, x=None, y=None, **kwds):
             A color or a Field name to draw the color of the marker from
         s : int, optional, also available as 'size'
             The size of the marker
-        by : string, optional
```

Comment:
I think these options were added by Marc when he improved all the docstrings. I understand why he did that, to make them available in VS Code when you use hvPlot as follows:
```
from hvplot import hvPlot

hvPlot(df).scatter(...
```

It duplicates docstrings which isn't good. Removing them affects the UX in an IDE, which isn't good either. I'll open an issue to track this, I think we could define a subset of important options all methods should have (or have a way to declare which generic options a given method should be augmented with in its docstring), and have a pre-commit step that makes sure we don't have to manually duplicate docstrings.